installation/installFromSource.md

```diff
@@ -2,13 +2,14 @@
 sort: 1
 ---
 
-# Installing solids4foam from source
+# Installing solids4foam
+Installing solids4foam-v2.0 includes two easy steps, i.e., 
+ - [*downloading* the source code](#downloading-the-solids4foam-source-code), and
+ -  [*compiling*](#compiling-solids4foam) it. 
 
```

Comment:
How we begin the installation instruction, as it is the first thing that a new user face, is the best opportunity to catch the user's attention!
The current instruction does not start friendly enough. I suggest the following
```suggestion
nstalling solids4foam-v2.0 includes two easy steps, i.e., 
 - [*downloading* the source code](#downloading-the-solids4foam-source-code), and
 - [*compiling*](#compiling-solids4foam) it. 

Before going through these steps, make sure you have a working version of OpenFOAM. If you have not OpenFOAM installed yet, we suggest downloading one of the OpenFOAM versions compatible with solids4foam, listed in the table below.
```